tests/python/test_elastic_net.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import unittest
+import xgboost as xgb
+
+from numpy.testing import assert_approx_equal
+
+train_data = xgb.DMatrix(np.array([[1]]), label=np.array([1]))
+
```

Comment:
This filename is a little confusing as we also have linear models in XGBoost. May be call it test_tree_regularization.py and test_alpha/test_lambda?